google-cloud-tools-plugin/src/com/google/gct/idea/debugger/UpdateIdeWithBreakpointAction.java

```diff
@@ -0,0 +1,73 @@
+package com.google.gct.idea.debugger;
+
+import com.google.api.client.repackaged.com.google.common.base.Strings;
+import com.google.api.services.clouddebugger.model.Breakpoint;
+
+import com.intellij.debugger.ui.breakpoints.BreakpointManager;
+import com.intellij.openapi.vfs.VirtualFile;
+import com.intellij.xdebugger.breakpoints.XBreakpoint;
+import com.intellij.xdebugger.breakpoints.XBreakpointManager;
+import com.intellij.xdebugger.breakpoints.XLineBreakpoint;
+
+import java.util.Map;
+
+class UpdateIdeWithBreakpointAction implements Runnable {
```

Comment:
It looks like Task is also a jetbrains concept
(com.intellij.openapi.progress).  I'm not full of great idea about this..
maybe DoUpdateIdeWithBreakpointAction?

On Mon, Jan 25, 2016 at 11:11 AM, akerekes <notifications@github.com> wrote:

> In
> google-cloud-tools-plugin/src/com/google/gct/idea/debugger/UpdateIdeWithBreakpointAction.java
> <https://github.com/GoogleCloudPlatform/gcloud-intellij/pull/375#discussion_r50714516>
> :
>
> > @@ -0,0 +1,73 @@
> > +package com.google.gct.idea.debugger;
> > +
> > +import com.google.api.client.repackaged.com.google.common.base.Strings;
> > +import com.google.api.services.clouddebugger.model.Breakpoint;
> > +
> > +import com.intellij.debugger.ui.breakpoints.BreakpointManager;
> > +import com.intellij.openapi.vfs.VirtualFile;
> > +import com.intellij.xdebugger.breakpoints.XBreakpoint;
> > +import com.intellij.xdebugger.breakpoints.XBreakpointManager;
> > +import com.intellij.xdebugger.breakpoints.XLineBreakpoint;
> > +
> > +import java.util.Map;
> > +
> > +class UpdateIdeWithBreakpointAction implements Runnable {
>
> Yeah, I was contemplating on it as well, but calling it Runnable would
> have been redundant, Action seemed a bit better. Maybe
> UpdateIdeWithBreakpointTask?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/gcloud-intellij/pull/375/files#r50714516>
> .
>